web/modules/custom/mymodule/mymodule.module

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Module file of custom module.
+ */
+
+/**
+ * Function to alter the module to restrict the page.
+ *
+ * @param object $form
+ *   It will give the $form data.
+ * @param FormStateInterface $form_state
+ *   It will retrieve the form values.
+ * @param object $form_id
+ *   It will return the form_id.
+ */
+function mymodule_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
@anitha-innoraft - What is the purpose of this form_alter?

Reply:
After creating the custom permission. I have tried to check the permission of form using form alter as suggested in forum using the above code.

Reply:
Let's remove this @anitha-innoraft - this would cause unintended issues.

Reply:
Removed the code with that file. Please check once.